addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -365,7 +365,7 @@ def _l10n_es_tbai_get_invoice_values(self, invoice, cancel):
         export_exempts = invoice.invoice_line_ids.tax_ids.filtered(lambda t: t.l10n_es_exempt_reason == 'E2')
         values['regime_key'] = ['02'] if export_exempts else ['01']
 
-        if invoice.l10n_es_is_simplified:
+        if invoice.l10n_es_is_simplified and invoice.company_id.l10n_es_tbai_tax_agency != 'bizkaia':
             values['regime_key'].append(52)  # code for simplified invoices
```

Comment:
```suggestion
            values['regime_key'] += ['52']  # code for simplified invoices
```
I know it has nothing to do with the current changes, but it bothers me that we mix strings and integers :nerd_face: